source/atlas-cli-create-cluster-from-config-file.txt

```diff
@@ -0,0 +1,225 @@
+.. _atlas-cli-create-cluster-from-config-file:
+
+======================================================
+Create an |service| Cluster Using a Configuration File
+======================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This tutorial demonstrates how to use {+atlas-cli+} commands to create
+a new |service| cluster from a configuration file. Specifically, it
+demonstrates how to:
+
+1. Get the configuration settings of an existing |service| {+cluster+}
+   and save the settings to a configuration file using the
+   :ref:`atlas-clusters-describe` command. 
+#. Create an |service| {+cluster+} from the configuration file using the
+   :ref:`atlas-clusters-create` command.
+
+.. _atlas-cli-create-cluster-from-config-file-reqs:
+
+Prerequisites 
+-------------
+
+Before you begin, you must have the following: 
+
+- An |service| {+cluster+} 
+- :ref:`Atlas CLI <install-atlas-cli>` 
+- :ref:`A profile <atlas-cli-profiles>` that contains the following: 
+
+  - IDs of the |service| organization and project from where you wish to
+    retrieve existing cluster settings and where you wish to create the
+    new {+cluster+}. 
+  - |api| keys for programmatic access.
+
+.. _atlas-cli-create-cluster-from-config-file-procedure:
+
+Create an |service| Cluster From a Configuration File 
+-----------------------------------------------------
+
+You can use the procedures in this section to easily create a new
+{+cluster+} by exporting settings from an existing {+cluster+} instead
+of manually creating a configuration file with necessary {+cluster+}
+configuration settings for creating a new {+cluster+}. 
+
+Export Existing Cluster Configuration Settings to a File 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Connect to your |service| account for programmatic access. 
```

Comment:
[q] this is already part of the per-requisites, is it an issue to duplicate it?

Reply:
I don't think so. It is probably ok to repeat it to make sure they have to connect with the profile to run the commands (especially if the user is using an existing profile and doesn't bother to click the link in the prerequisites).